coursier.bzl

```diff
@@ -815,7 +822,8 @@ coursier_fetch = repository_rule(
             default = False,
         ),
         "resolve_timeout": attr.int(default = 600),
-        "jetify": attr.bool(doc = "Runs the AndroidX Jetifier tool on all artifacts.", default = False)
+        "jetify": attr.bool(doc = "Runs the AndroidX Jetifier tool on all artifacts.", default = False),
```

Comment:
```suggestion
        "jetify": attr.bool(doc = "Runs the AndroidX Jetifier tool on artifacts specified in jetify_include_list. If jetify_include_list is not specified, run jetifier on all artifacts.", default = False),
```

here and above.

Reply:
Nit: use either `jetifier` or Jetifier in docs.

Also think about providing URL to the Jetifier page (Stardoc supports Markdown).